pmm-app/tests/pages/homePage.js

```diff
@@ -14,25 +14,23 @@ module.exports = {
     dashboardHeaderText: 'Percona Monitoring and Management',
     dashboardHeaderLocator: "//div[contains(@class, 'dashboard-header')]",
     checkUpdateButton: "#refresh",
+    oldLastCheckSelector: "#pmm-update-widget > .last-check-wrapper p",
     lastCheckSelector: ".last-check-wrapper > p",
     triggerUpdate: "button[ng-click='update()']",
     updateProgressModal: ".modal-content",
     reloadButtonAfterUpgrade: "button[ng-click='reloadAfterUpdate()'",
     pmmUpdateWidget: "#pmm-update-widget",
-    upToDateLocator: locate('p').withText('You are up to date').inside('.state'),
+    upToDateLocator: "//section[@class='state']/p[text()='You are up to date']",
     availableVersion: "#available_version > div > p",
     currentVersion: "#current_version > span",
     sttDisabledFailedChecksPanelSelector: "$db-check-panel-settings-link",
     sttFailedChecksPanelSelector: "$db-check-panel-has-checks",
     checksPanelSelector: "$db-check-panel-home",
-
+    newsPanelTitleSelector: "//span[@class='panel-title-text' and text() = 'Percona News']",
+    newsPanelContentSelector: "//span[contains(text(), 'Percona News')]/ancestor::div[contains(@class, 'panel-container')]//div[contains(@class, 'view')]"
```

Comment:
`div.view` hosts all of the Grafana panels. Do we really need to be that loose here?

Reply:
this `div.view` is coupled to the Percona News panel, that's why i used `ancestor`. This is the only way we can check it for now